src/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{E0C4E267-B1D6-463B-9C95-8C0D3C335924}</ProjectGuid>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetGroup)' == 'netstandard' AND '$(OSGroup)' == 'AnyOS'">SR.PlatformNotSupported_X509Certificate2UI</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Assuming this library will only ever get implemented in netcoreapp, the "and" clause here is redundant.